src/screens/WhotoFollow/index.js

```diff
@@ -31,104 +30,97 @@ import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
 import {Header} from '../../components';
 import {InitialStartupAtom} from '../../service/initialStartup';
 import {ProgressBar} from '../../components/ProgressBar';
-import {get} from '../../api/server';
 import {normalizeFontSize} from '../../utils/fonts';
 import {registerUser} from '../../service/users';
 import {setImage} from '../../context/actions/users';
 import {setToken} from '../../utils/token';
 import {useClientGetstream} from '../../utils/getstream/ClientGetStram';
+import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
+import {getWhoToFollowList} from '../../service/topics';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `getWhoToFollowList` import from `../../service/topics` shows that the developer has moved the logic for fetching the "who to follow" list into a separate service function. This is a good practice as it improves modularity and makes the code easier to test and maintain.
```diff
+ import {getWhoToFollowList} from '../../service/topics';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->